clients/drcachesim/common/trace_entry.h

```diff
@@ -300,6 +308,9 @@ typedef enum {
      * The marker value contains the return value of the just-entered function,
      * whose id is specified by the closest previous #TRACE_MARKER_TYPE_FUNC_ID
      * marker entry
+     *
+     * The marker value for system calls (see #TRACE_FUNC_ID_SYSCALL_BASE) is either 0
+     * (failure) or 1 (success).
```

Comment:
Add the caveats to this "success" code here (i#5843).

Reply:
Added links to the DR docs about this.